hadoop-ozone/dist/src/main/smoketest/upgrade/snapshot.robot

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Currently the smoketest/upgrade directory has tests for common upgrade functionality, and feature specific upgrade tests are in their own directory with upgrade specific names. See `upgrade-ec-check.robot` for an example. We should try to keep that layout consistent here too.

Reply:
Yup sure! moved to snapshot's own robot test directory with upgrade specific names